packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -328,6 +330,13 @@ export const pivotQueryResults = ({
                 )
             ) {
                 columnCount++;
+
+                if (columnCount > MAX_COLUMNS) {
+                    throw new Error(
+                        `Cannot pivot results with more than ${MAX_COLUMNS} columns`,
+                    );
+                }
```

Comment:
Something above this is catching everything?

Reply:
yeah, forgot to include screenshot in the description: (updating)

<img width="933" alt="CleanShot 2023-08-30 at 18 29 25@2x" src="https://github.com/lightdash/lightdash/assets/962095/8dcc1fd9-b06d-4359-949b-96186029e83c">

Reply:
TODO: add more

Reply:
What about `Try adding a filter to limit your results.`